tests/test_parametric_shapes/test_RotateMixedShape.py

```diff
@@ -181,6 +181,31 @@ def test_export_stl(self):
         assert Path("filename.stl").exists() is True
         os.system("rm filename.stl")
 
+    def test_export_stp(self):
```

Comment:
There is another test_export_stp in this file.

Reply:
renamed the method